src/main/java/uk/gov/ida/verifyserviceprovider/factories/saml/AuthnRequestFactory.java

```diff
@@ -84,6 +103,18 @@ public AuthnRequest build(LevelOfAssurance levelOfAssurance, String serviceEntit
         return authnRequest;
     }
 
+    private RequestedAuthnContext createRequestedAuthnContext(List<String> authnContexts) {
+        RequestedAuthnContext requestedAuthnContext= new RequestedAuthnContextBuilder().buildObject();
+        List<AuthnContextClassRef> authnContextClassRefs = authnContexts.stream().map((a) -> {
+            AuthnContextClassRef authnContextClassRef = new AuthnContextClassRefBuilder().buildObject();
+            authnContextClassRef.setAuthnContextClassRef(a);
+            return authnContextClassRef;
+        }).collect(Collectors.toList());
+        requestedAuthnContext.getAuthnContextClassRefs().addAll(authnContextClassRefs);
+        requestedAuthnContext.setComparison(AuthnContextComparisonTypeEnumeration.EXACT);
```

Comment:
Is this the right compaision type? I thought it was supposed be `minimum`. Is there a spec somewhere from when we decided this 2(?) years ago.

Reply:
`minimum` was the legacy comparison type and `exact` the new one. At the moment, only IDPs care about the comparison type, hence why we added a couple of new IDP tests to compliance tool to test it (see commit `fc58f88a01ab8f100122337b67520c68991782c8`).

Reply:
The reason I ask is I remember us saying that we want to use a different comparison type for the SPs and IDPs.